ci/test-11-unpriv.pl

```diff
@@ -40,7 +40,7 @@ sub get_ping_gid_range {
 }
 
 sub test_unprivileged_works {
-    plan tests => 12;
+    plan tests => 13;
```

Comment:
You have added 3 tests, not one (see above for the explanation).

Reply:
Changed to `15`.